pkg/kritis/review/validating_transport_test.go

```diff
@@ -237,29 +237,35 @@ func TestValidatingTransport(t *testing.T) {
 		{name: "invalid secret", auth: validAuthWithOneGoodPgpKey, wantAtts: []attestation.ValidatedAttestation{}, attestations: []metadata.RawAttestation{
 			metadata.MakeRawAttestation(metadata.PgpSignatureType, encodeB64("invalid-sig"), "invalid-fpr", ""),
 		}, errorExpected: false, attError: nil},
-		{name: "valid sig over another host", auth: validAuthWithOneGoodPgpKey, wantAtts: []attestation.ValidatedAttestation{}, attestations: []metadata.RawAttestation{
+		// TODO(acamadeo): After payload check is implemented in cryptolib,
+		// `wantAtts` should be an empty list.
+		{name: "valid sig over another host", auth: validAuthWithOneGoodPgpKey, wantAtts: []attestation.ValidatedAttestation{
```

Comment:
This actually changes the behavior of Kritis. Can we avoid it?

Reply:
Reverted to original behavior.